save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestService.kt

```diff
@@ -34,19 +39,34 @@ class TestService(private val configProperties: ConfigProperties) {
     @Autowired
     private lateinit var testExecutionRepository: TestExecutionRepository
 
+    @Autowired
+    private lateinit var testSuiteRepository: TestSuiteRepository
+
     /**
      * @param tests
+     * @return list tests id's
      */
-    fun saveTests(tests: List<Test>) {
-        testRepository.saveAll(tests)
+    fun saveTests(tests: List<TestDto>): List<Long> {
+        val testsId: MutableCollection<Long> = mutableListOf()
+        tests.map { testDto ->
```

Comment:
You should either use `forEach`, because mapped list is useless here, or return `id` instead of adding it to a mutable list